CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing To JobSeeker
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to JobSeeker and its packages, which are hosted in the [RedHat-Beyond](https://github.com/redhat-beyond) Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+You are welcome to contact us through slack, with link #os-07-group4 on [beyond slack](redhat-beyond.slack.com).
```

Comment:
Consider adding contact header.
Please refer this contribution guide as sample
https://docs.openstack.org/neutron/latest/contributor/contributing.html

Reply:
I have updated the file in [6213999](https://github.com/redhat-beyond/JobSeeker/commit/621399924ef0c55d7971ff73ef460f7273254403). You may review.